linkerd/app/src/env.rs

```diff
@@ -500,6 +507,18 @@ pub fn parse_config<S: Strings>(strings: &S) -> Result<super::Config, EnvError>
     })
 }
 
+fn convert_string_to_map(labels: String) -> HashMap<String, String> {
```

Comment:
nit: maybe the name should suggest that this is specifically converting pod labels for OpenCensus? `convert_string_to_map` could be converting any string to any map...